README.md

```diff
@@ -103,57 +103,126 @@ FLP is a framework. The main FLP object is the **pipeline**. FLP **pipeline** ca
 
 # Architecture
 
-The pipeline is constructed of a sequence of stages:
+The pipeline is constructed of a sequence of stages. Each stage is classified into one of the following classes:
```

Comment:
Do we use the term class or do we use `type` ... I do not want to confuse the readers